lib/moon/assets/age_limit/age18_small.ex

```diff
@@ -17,7 +17,13 @@ defmodule Moon.Assets.AgeLimit.Age18Small do
   def render(assigns) do
     ~F"""
     <svg
-      class={"moon-age_limit #{@class} #{@click && "cursor-pointer"}"}
+      class={
+        "moon-age_limit",
+        @class,
+        "text-#{@color}": @color,
```

Comment:
Please add lines, so if class is "text-gohan-100" then it will work also.

`
"moon-age_limit",
        @class,
        @color
        "text-#{@color}": @color,
`